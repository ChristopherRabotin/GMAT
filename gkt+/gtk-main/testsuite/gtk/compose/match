<Multi_key> <s> <e> <q> : "!"
<Multi_key> <s> <e> <q> <u> : "?"
<Multi_key> <z> <w> <i> <n> <e> <s> : "ðŸ¥‚"
<Multi_key> <l> <o> <n> <g> : "this is a long replacement string"
