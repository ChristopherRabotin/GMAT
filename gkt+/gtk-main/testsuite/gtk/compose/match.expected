# n_sequences: 4
# max_seq_len: 7
# n_index_size: 1
# data_size: 29
# n_chars: 39
<Uff20> <U73> <U65> <U71> : "!" # U21
<Uff20> <U6c> <U6f> <U6e> <U67> : "this is a long replacement string"
<Uff20> <U73> <U65> <U71> <U75> : "?" # U3f
<Uff20> <U7a> <U77> <U69> <U6e> <U65> <U73> : "ðŸ¥‚" # U1f942
